bin/parse_sequenceQC.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import json
+import csv
+import argparse
+from pathlib import Path
+import gzip
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Extract FAILED QC messages from mikro JSON file"
+    )
+    parser.add_argument(
+        "-i", "--input", required=True, type=Path,
+        help="Path to the mikro QC JSON file (.json or .json.gz)"
```

Comment:
Recommend: "Path to the Mikrokondo-generated QC JSON output file.", unless "mikro" has a specific meaning that I don't know about.